system_meta/views.py

```diff
@@ -0,0 +1,23 @@
+"""Views for the REST API for system metadata."""
+
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from system_meta.models import IntegratedSystem, Product
+from system_meta.serializers import IntegratedSystemSerializer, ProductSerializer
+
+
+class IntegratedSystemViewSet(viewsets.ModelViewSet):
+    """Viewset for IntegratedSystem model."""
+
+    queryset = IntegratedSystem.objects.all()
+    serializer_class = IntegratedSystemSerializer
+    permission_classes = (IsAuthenticated,)
+
+
+class ProductViewSet(viewsets.ModelViewSet):
```

Comment:
These don't have pagination setup on them and we want to avoid creating that kind of situation right off the bat. I think since this is somewhat of a greenfield app we should just set the default globally: https://www.django-rest-framework.org/api-guide/pagination/#setting-the-pagination-style